rsts/user/getting_started/create_first.rst

```diff
@@ -37,89 +34,61 @@ The most basic Flyte primitive is a "task". Flyte Tasks are units of work that c
+* Use the ``@task`` decorator to convert your typed python function to a Flyte task.
+* A ``FlyteFile`` is a Flyte Kit type that represents binary data.  It is used to offload data to a storage location like S3.  Here we use it to store an image.
 
 Writing a Workflow
 *********************
 Next you need to call that task from a workflow.  In the same file, add these lines.
 
 .. code-block:: python
 
-  @workflow_class
-  class EdgeDetectorWf(object):
-      image_input = Input(Types.String, required=True, help="Image to run for")
-      run_edge_detection = edge_detection_canny(image_location=image_input)
-      edges = Output(run_edge_detection.outputs.parsed_image, sdk_type=Types.Blob)
-  
+   @workflow
+   class EdgeDetectorWf(image_input: str) -> FlyteFile:
+       edges = edge_detection_canny(image_location=image_input)
+       return edges
+
 This code block creates a workflow, with one task. The workflow itself has an input (the link to an image) that gets passed into the task, and an output, which is the processed image.
 
+You can call this workflow ``EdgeDetectorWf(image_input=...)`` and iterate locally before moving on to register it with Flyte.
 
 Interacting with Flyte
 ************************
 
-Flyte fulfills tasks using docker images. You'll need to build a docker image from this code before it can run in Flyte. The repo has a make target to build the docker image for you ::
-
-  eval $(minikube -p minikube docker-env)  # If using the flyte sandbox via minikube
-  make docker_build
-
-If you have the flyte sandbox installed on your local machine, the image will be accessible to your Flyte system. If you're running a remote Flyte instance, you'll need to upload this image to a remote registry such as Dockerhub, Amazon ECR, or Google Container Registry, so that it can be used by the Flyte system.
-
-To upload to a remote registry (or even local registry), use ::
-
-  DOCKER_REGISTRY_USERNAME={username} DOCKER_REGISTRY_PASSWORD={pass} REGISTRY=docker.io make docker_build
-
-Replace the values above with your registry username, password, and registry endpoint.
-
-You may need to change the ``IMAGE_NAME`` in the Makefile to reflect your namespace in the docker registry. (ex ``{{my docker username}}/myflyteproject``)
-
-With the image built, we just need to register the tasks and workflows. The process is the same as what we had done previously. ::
-
-  docker run --network host -e FLYTE_PLATFORM_URL='127.0.0.1:30081' {{ your docker image }} pyflyte -p myflyteproject -d development -c sandbox.config register workflows
-
-After this, you should be able to visit the Flyte UI, and run the workflow as you did with ``flytesnacks`` previously.
+For detailed and interactive example workflow 'recipes' check out the `Flytesnacks Cookbook <https://flytecookbook.readthedocs.io/en/latest/>`__
```

Comment:
Dive deeper and try out more examples then look into. Also I think this should be the first `getting started` page, the one about installing flyte etc should come after this

Thus you might want to say, 
`Intrigued, want to learn more and try more complex examples, cookbook link`

Reply:
done.